api/lib/domain/services/mail-service.js

```diff
@@ -26,7 +26,8 @@ function sendOrganizationInvitationEmail({
   email,
   organizationName,
   organizationInvitationId,
-  code
+  code,
+  tags
```

Comment:
Titre de commit
Use tags property in mail service  => Add tags when sending invitation in mail service

Reply:
Done